services/azure-servicebus/src/main/java/com/microsoft/windowsazure/services/servicebus/ServiceBusContractAsync.java

```diff
@@ -0,0 +1,578 @@
+package com.microsoft.windowsazure.services.servicebus;
+
+import com.microsoft.windowsazure.core.pipeline.jersey.JerseyFilterableService;
+import com.microsoft.windowsazure.exception.ServiceException;
+import com.microsoft.windowsazure.services.servicebus.models.BrokeredMessage;
+import com.microsoft.windowsazure.services.servicebus.models.CreateQueueResult;
+import com.microsoft.windowsazure.services.servicebus.models.CreateRuleResult;
+import com.microsoft.windowsazure.services.servicebus.models.CreateSubscriptionResult;
+import com.microsoft.windowsazure.services.servicebus.models.CreateTopicResult;
+import com.microsoft.windowsazure.services.servicebus.models.GetQueueResult;
+import com.microsoft.windowsazure.services.servicebus.models.GetRuleResult;
+import com.microsoft.windowsazure.services.servicebus.models.GetSubscriptionResult;
+import com.microsoft.windowsazure.services.servicebus.models.GetTopicResult;
+import com.microsoft.windowsazure.services.servicebus.models.ListQueuesOptions;
+import com.microsoft.windowsazure.services.servicebus.models.ListQueuesResult;
+import com.microsoft.windowsazure.services.servicebus.models.ListRulesOptions;
+import com.microsoft.windowsazure.services.servicebus.models.ListRulesResult;
+import com.microsoft.windowsazure.services.servicebus.models.ListSubscriptionsOptions;
+import com.microsoft.windowsazure.services.servicebus.models.ListSubscriptionsResult;
+import com.microsoft.windowsazure.services.servicebus.models.ListTopicsOptions;
+import com.microsoft.windowsazure.services.servicebus.models.ListTopicsResult;
+import com.microsoft.windowsazure.services.servicebus.models.QueueInfo;
+import com.microsoft.windowsazure.services.servicebus.models.ReceiveMessageOptions;
+import com.microsoft.windowsazure.services.servicebus.models.ReceiveMessageResult;
+import com.microsoft.windowsazure.services.servicebus.models.ReceiveQueueMessageResult;
+import com.microsoft.windowsazure.services.servicebus.models.ReceiveSubscriptionMessageResult;
+import com.microsoft.windowsazure.services.servicebus.models.RuleInfo;
+import com.microsoft.windowsazure.services.servicebus.models.SubscriptionInfo;
+import com.microsoft.windowsazure.services.servicebus.models.TopicInfo;
+import com.sun.jersey.api.client.ClientResponse;
+
+import rx.Observable;
+
+public interface ServiceBusContractAsync extends
+JerseyFilterableService<ServiceBusContract> {
+
+/**
+* Sends a queue message.
+* 
+* @param queuePath
+*            A <code>String</code> object that represents the name of the
+*            queue to which the message will be sent.
+* @param message
+*            A <code>Message</code> object that represents the message to
+*            send.
+* @throws ServiceException
```

Comment:
I was a bit wrong. Actually this should go to the @returs section: this method doesn't throw but returned Observable may throw this exception